### PR TITLE
Fix start and end question to send accurate data

### PR DESCRIPTION
**Pull request by blahajjessie**

### CODE REVIEW

File: api-docs/interface/server-socket.md [hunk 1]
```diff
@@ -50,7 +51,7 @@ Fields:
 - `scoreChange` (number) = how much their score increased due to this question, The host will recieve garbage
 - `correct` (boolean) : if the player's answer to the question is correct. The host will recieve garbagae
 - `leaderboard`: sorted array of [{name: string, score: number }, …]
-- `time` : number: the ammount of time the player took to answer the question
+- `responseTime` : number: the ammount of time the player took to answer the question
```
Comment (190n) [inline api-docs/interface/server-socket.md]: Should also be fixed in #34 but this should specify milliseconds

File: api-docs/interface/server-socket.md [hunk 2]
```diff
@@ -61,6 +62,7 @@ It will contain a player id and username that has completed the most recent acti
 This may be sent many times rapidly on the case of a host just conneccting. 
 
 Fields: 
-- `player`: {`id`, `username`}
+- `id` (userId): the player's user ID
+- `username` (string): the name of the player
```
Comment (190n) [inline api-docs/interface/server-socket.md]: apply the fix from #34

**Review by 190n — CHANGES_REQUESTED** [review]
`time` is not fixed on `startQuestion` messages.

**Comment by 190n** [comment]
`correctAnswers` is not sent on `endQuestion` messages to host.

File: api-docs/interface/player-http.md [hunk 3]
```diff
@@ -1,12 +1,16 @@
 # Game Host (teacher) API's
 
 ### Join game
-POST /games/:gameId/players: join a game
-body: {name: string}
+`POST /games/:gameId/players`: join a game
+body: `{name: string}`
```
Comment (190n) [inline api-docs/interface/player-http.md]: username

File: api-docs/interface/server-socket.md [hunk 4]
```diff
@@ -35,32 +39,46 @@ Sent by server whenever a new question is available
...
+- `score` (number) = the player’s current score, the host will receive garbage
+- `scoreChange` (number) = how much their score increased due to this question, The host will receive garbage
+- `correct` (boolean) : if the player's answer to the question is correct. The host will receive garbage
+- `leaderboard`: sorted array of [{name: string, score: number }, …] to represent the player score
+- `responseTime` (number) : the amount of time the player took to answer the question (ms)
+- `questionText` (string) : The question
+- `answerTexts` (string[]) : string[] of the answer choices
+- `index` (number) : the number of the question that just ended
+- `username` (string) : the name of the player
+- `explanations` (string[]) : The explanations for right ans wrong answers. TODO (Currently sends an empty array)
```
Comment (190n) [inline api-docs/interface/server-socket.md]: String array or null if there aren't explanations

**Review by 190n — CHANGES_REQUESTED** [review]

**Review by 190n — DISMISSED** [review]

**Review by 190n — DISMISSED** [review]

File: api-docs/interface/host-http.md [hunk 5]
```diff
@@ -1,51 +1,72 @@
...
 Description: Host creates a game
-Server sends response of: {gameId: string, hostId: string}
+
+response: `{gameId: string, hostId: string}`
 
 The game opens on the server side, and is now accepting players via the join code (which is the game ID)
-Upon recept of this, the host should display a screen showing a list of players.
 
-### Start a question
+Upon recept of this, the host should display a screen showing a list of players.
```
Comment (celdond) [inline api-docs/interface/host-http.md]: Typo, what is 'recept' ?

**Review by rlchoy — DISMISSED** [review]

File: api-docs/interface/player-http.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
+# Game Player (Student) API's
+
+## Join game
+
+#### Description: 
+Allows a player ot join a game
```
Comment (celdond) [inline api-docs/interface/player-http.md]: Typo
```suggestion
Allows a player to join a game
```

**Review by celdond — CHANGES_REQUESTED** [review]
Typos, there are also failing tests.  Not sure what branch the failures are from however.

File: api-docs/interface/host-http.md (hunk 5 above)
Comment (190n) [reply api-docs/interface/host-http.md]: probably "receipt". @blahajjessie

**Review by celdond — APPROVED** [review]

**Review by 190n — APPROVED** [review]

**Pull request merged**